core/src/main/java/org/infinispan/container/versioning/NumericVersionGenerator.java

```diff
@@ -36,35 +26,18 @@
  */
 @Scope(Scopes.NAMED_CACHE)
 public class NumericVersionGenerator implements VersionGenerator {
-
-   private static final Log log = LogFactory.getLog(NumericVersionGenerator.class);
-
    // TODO: Possibly seed version counter on capped System.currentTimeMillis, to avoid issues with clients holding to versions in between restarts
    final AtomicInteger versionCounter = new AtomicInteger();
-   final AtomicLong versionPrefix = new AtomicLong();
    private static final NumericVersion NON_EXISTING = new NumericVersion(0);
 
    @Inject Configuration configuration;
```

Comment:
I'd rather not do that, because then I'd need a factory.
If the constructor has no parameters, I can use `EmptyConstructorFactory` (and I actually have a pet peeve with components without an empty constructor being instantiated by `EmptyConstructorFactory` or `EmptyConstructorNamedCacheFactory`).